.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+yarn lint
```

Comment:
we'd like to run lint-staged + prettier as well as a part of the pre-commit right?

Reply:
Okay, will add them asap.